docs/UserGuide.adoc

```diff
@@ -17,161 +17,83 @@ endif::[]
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+=== Adding an expense: `add expense`
 
-[TIP]
-A person can have any number of tags (including 0)
+Format: `add expense n/EXPENSE_NAME c/COST [t/TAG]...`
 
 Examples:
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+*   `add expense n/Laksa c/10.50 t/Lunch t/Food`
+*   `add expense n/Travel c/53 t/Travel`
 
-=== Listing all persons : `list`
+Expected output:
 
-Shows a list of all persons in the address book. +
-Format: `list`
+*   `New entry added: 1. [-] Laksa ($10.50) [Lunch][Food]`
+*   `New entry added: 2. [-] Travel ($53) [Travel]`
 
-=== Editing a person : `edit`
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+=== Adding an income: `add income`
 
-****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
-****
+Format: `add income n/INCOME_NAME c/COST [t/TAG]...`
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@example.com` +
-Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-* `edit 2 n/Betsy Crower t/` +
-Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*   `add income n/Allowance c/1000 t/Monthly`
 
-=== Locating persons by name: `find`
+Expected output:
 
-Finds persons whose names contain any of the given keywords. +
-Format: `find KEYWORD [MORE_KEYWORDS]`
+*   `New entry added: 3. [+] Allowance ($1000) [Monthly]`
 
-****
-* The search is case insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
-****
+=== Deleting a Transaction: `delete`
 
-Examples:
-
-* `find John` +
-Returns `john` and `John Doe`
-* `find Betsy Tim John` +
-Returns any person having names `Betsy`, `Tim`, or `John`
-
-// tag::delete[]
-=== Deleting a person : `delete`
-
-Deletes the specified person from the address book. +
-Format: `delete INDEX`
-
-****
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index *must be a positive integer* 1, 2, 3, ...
-****
+Format: `delete i/EXPENSE_INDEX`
```

Comment:
Added in commit [36d1c8b](https://github.com/AY1920S1-CS2103T-W12-2/main/pull/51/commits/36d1c8b7700aaff74b88d857e5d89e685ea1620c).

Reply:
Are we retaining the index number of each transaction in the full transaction list after users have filtered by tags? @kensonoenqh

If not, this could lead to incorrect deletion of transaction.

Reply:
I think we should use the list currently displayed in the UI. Fixed the wording in commit 06e6a0c.